vllm/engine/metrics.py

```diff
@@ -44,6 +47,22 @@ def add_global_metrics_labels(**kwargs):
     "vllm:cpu_cache_usage_perc",
     "CPU KV-cache usage. 1 means 100 percent usage.")
 
+histogram_request_prompt_tokens = Histogram(
+    "vllm:request_prompt_tokens",
+    "Number of prefill tokens processed.",
+    buckets=[
+        1, 2, 5, 10, 20, 50, 100, 200, 500, 1_000, 2_000, 5_000, 10_000,
+        20_000, 50_000, 100_000
+    ],
+)
+histogram_request_generation_tokens = Histogram(
+    "vllm:request_generation_tokens",
+    "Number of generation tokens processed.",
+    buckets=[
+        1, 2, 5, 10, 20, 50, 100, 200, 500, 1_000, 2_000, 5_000, 10_000,
+        20_000, 50_000, 100_000
+    ],
+)
```

Comment:
Should we consider the value of  [`max_model_len`](https://github.com/vllm-project/vllm/blob/c9b45adeeb0e5b2f597d1687e0b8f24167602395/vllm/config.py#L50-L51) when setting the buckets? IIUC, `max_model_len` is the upper bound.